ebpf/monitor/accesslog.proto

```diff
@@ -0,0 +1,340 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+syntax = "proto3";
+
+package skywalking.v3;
+
+option java_multiple_files = true;
+option java_package = "org.apache.skywalking.apm.network.ebpf.monitor.accesslog.v3";
+option go_package = "skywalking.apache.org/repo/goapi/collect/ebpf/monitor/accesslog/v3";
+
+import "common/Common.proto";
+import "common/Command.proto";
+
+service EBPFAccessLogService {
+  rpc collect(stream EBPFAccessLogMessage) returns (EBPFAccessLogDownstream) {
+  }
+}
+
+message EBPFAccessLogMessage {
+  // current node information, only not null when first message or have update
+  EBPFAccessLogNodeInfo node = 1;
+  // local process and remote process connection information
+  AccessLogConnection connection = 2;
+  // kernel level metrics
+  repeated AccessLogKernelLog kernelLogs = 3;
```

Comment:
Is the kernel log with HTTP log?

Reply:
What is the period and moment of sending TCP relative logs?

Reply:
If we cannot detect the kernel related to the application logs, then they should be sent through a timer. 
Otherside, the kernel logs should related by a single application log.